ansible/roles/os_zabbix/vars/template_openshift_master.yml

```diff
@@ -340,6 +340,18 @@ g_template_openshift_master:
     url: 'https://github.com/openshift/ops-sop/blob/master/V3/Alerts/openshift_master.asciidoc'
     priority: info
 
+  - name: 'Failed Persistent Volumes on {HOST.NAME}'
+    expression: '{Template Openshift Master:openshift.master.pv.failed.count.last(#2)}>0'
+    priority: info
+
+  - name: 'Less than 10 Available Persistent Volumes on {HOST.NAME}'
+    expression: '{Template Openshift Master:openshift.master.pv.available.count.last(#2)}<10'
+    priority: info
+
+  - name: 'Less than 5 Available Persistent Volumes on {HOST.NAME}'
+    expression: '{Template Openshift Master:openshift.master.pv.available.count.last(#2)}<5'
```

Comment:
I would do `.max(#2) < 5` instead of .last. I try to not use `.last`. `.max` will helps with flapping better.

Unless there's a reason `.last` is better in this specific case.